packages/flex-dev-utils/package.json

```diff
@@ -32,23 +32,36 @@
   },
   "dependencies": {
```

Comment:
Having one location to have all of our common modules makes it easy to bump the version once, and have all the other repos automatically get the update.